Applications/Spire/Include/Spire/Ui/TabWidget.hpp

```diff
@@ -9,12 +9,30 @@ namespace Spire {
   class TabWidget : public QTabWidget {
     public:
 
-      //! Constructs a TabWidget.
+      //! Represents the supported tab content padding styles.
+      enum class PaddingStyle {
+
+        //! The content has no padding.
+        NONE,
+
+        //! The default content padding.
```

Comment:
Don't name an enum ```DEFAULT```, instead give it a name ```FOO``` that describes what it does and then the default value (for example in the constructor) can be use ```FOO```.